Model/Media.php

```diff
@@ -12,8 +12,8 @@
 namespace Sonata\MediaBundle\Model;
 
 use Imagine\Image\Box;
-use Sonata\ClassificationBundle\Model\CategoryInterface;
 use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
```

Comment:
I was referring to \UnexpectedValueException, but please use `\InvalidArgumentException` instead : http://fr2.php.net/manual/fr/class.invalidargumentexception.php it is more appropriate. No symfony validation please.